### DISCUSSION TITLE
Do Legacy cards have acceleration on Big Sur with this patch?

### DISCUSSION THREAD

**Discussion by ghost** [chris1111/Legacy-Video-patch#11] [Q&A]
Do Legacy cards have acceleration on Big Sur with this patch?

**Comment by chris1111** [comment]
> Do Legacy cards have acceleration on Big Sur with this patch?

no

**Reply by ghost** [reply]
I take it's because of the lack of OpenGL?

**Reply by chris1111** [reply]
> I take it's because of the lack of OpenGL?


No its a patch for Catalina

**Reply by ghost** [reply]
ok. sorry

**Reply by chris1111** [reply]
> ok. sorry

don't be sorry 🙂 it's correct it's just that the script of this program doesn't work in Big Sur, it might have to be changed to be able to test it on a dsk read write snapshot

**Reply by ghost** [reply]
If you'd like, I can take a look at it and send any necessary changes. I have a 2011 MBP running Big Sur.

**Reply by ghost** [reply]
I can also test the program on my new hackintosh, but that doesn't have a legacy card.

**Reply by chris1111** [reply]
> If you'd like, I can take a look at it and send any necessary changes. I have a 2011 MBP running Big Sur.

Your only have to change the script here https://github.com/chris1111/Legacy-Video-patch/blob/master/Sources/POST

I believe that adding this would be sufficient if you have an Intel HD 3000 card
![Capture d’écran, le 2020-12-26 à 08 07 23](https://user-images.githubusercontent.com/6248794/103152017-94d76800-4751-11eb-919e-37f1008dd496.png)

But not really sure because I not test BS on Legacy Video system with the old framework???
EDIT****
If com.apple.security.libraryvalidation.plist is need to = TRUE
Adding the Big Sur productVersion at the start of the script
if [[ $(sw_vers -productVersion) == "10.16" ]]; then
    echo "Latest Big Sur"
defaults write /Library/Preferences/com.apple.security.libraryvalidation.plist DisableLibraryValidation -bool true
echo "Install plist"
fi

One more thing I don't know?

**Reply by twitchplaystrading** [reply]
> If com.apple.security.libraryvalidation.plist is need to = TRUE
> Adding the Big Sur productVersion at the start of the script
> if [[ $(sw_vers -productVersion) == "10.16" ]]; then
> echo "Latest Big Sur"
> defaults write /Library/Preferences/com.apple.security.libraryvalidation.plist DisableLibraryValidation -bool true
> echo "Install plist"
> fi

Hi, just wanted to say thank you for your work chris. 

I tried the method above but i am faced with the error "runtime script error" at the end of the installation. I've edited the POST file with xcode and inserted your code and then created the app using the command file. Tried it on Big Sur 11.1 .

**Reply by tahaklkt-osxinfo** [reply]
@chris1111 hey bro are u working on big sur ? I and my friends waiting this patch for Big Sur. If you are not working on Big Sur, how can i use my nvidia gt 330m graphics card ?

**Reply by tahaklkt-osxinfo** [reply]
@chris1111 i installed legacy video patch to big sur. But it causes kernel panic. What can I do ?

**Reply by chris1111** [reply]
> @chris1111 i installed legacy video patch to big sur. But it causes kernel panic. What can I do ?


Nothing this patch is for Catalina
Closed topic
